chicago/templates/chicago/about.html

```diff
@@ -115,19 +115,30 @@ <h3 id='tags'>Legislation tags</h3>
 
-        <p>This data is then collected daily with the help of the <a href='http://webapi.legistar.com/' target="_blank">Legistar Web API</a> and stored using the <a href="http://opencivicdata.org/about/" target="_blank">Open Civic Data</a> standard and platform. Open Civic Data standardizes information about people, organizations, events and bills at any level of government and was built in collaboration with <a href="http://sunlightfoundation.com/" target="_blank">The Sunlight Foundation</a>, <a href="https://developers.google.com/civic-information/?hl=en" target="_blank">Google</a>, <a href="http://www.granicus.com/" target="_blank">Granicus</a> and <a href="http://www.opennorth.ca/" target="_blank">Open North</a>.
-        </p>
+        <p>Daily, DataMade collects data from Legistar and the <a href='http://webapi.legistar.com/' target="_blank">Legistar Web API</a>, which we store using the <a href="http://opencivicdata.org/about/" target="_blank">Open Civic Data</a> standard and platform. Built in collaboration with <a href="http://sunlightfoundation.com/" target="_blank">The Sunlight Foundation</a>, <a href="https://developers.google.com/civic-information/?hl=en" target="_blank">Google</a>, <a href="http://www.granicus.com/" target="_blank">Granicus</a>, and <a href="http://www.opennorth.ca/" target="_blank">Open North</a>, Open Civic Data standardizes information about people, organizations, events, and bills at any level of government.</p>
 
         <p><a href='http://opencivicdata.org' target="_blank"><img style='height:100px;' src='/static/images/ocd.jpg' /></a></p>
+
+        <p>Anyone can use the OCD API! This searchable government-transparency tool conatins several useful endpoints:</p>
+        <ul>
+            <li><a href="https://ocd.datamade.us/bills/" target="_blank">bills</a></li>
+            <li><a href="https://ocd.datamade.us/events/" target="_blank">events</a></li>
+            <li><a href="https://ocd.datamade.us/people/" target="_blank">people</a></li>
+            <li><a href="https://ocd.datamade.us/organizations/" target="_blank">organizations</a></li>
+            <li><a href="https://ocd.datamade.us/votes/" target="_blank">votes</a></li>
+        </ul>
+        <p>Want to see all bills from Chicago City Council? Add the "name" filter, and <a href='https://ocd.datamade.us/bills/?from_organization__name=Chicago%20City%20Council' target='_blank'>explore the results</a>. To learn more, read <a href='http://ocd-api-documentation.readthedocs.io/en/latest/index.html' target='_blank'>our documentation of the API</a> and its system of queryable endpoints.</p>
```

Comment:
lets make the link to the documentation more prominent (perhaps above the list of endpoints